src/data/sym/basic.lean

```diff
@@ -314,6 +314,54 @@ multiset.mem_attach _ _
   :=
 coe_injective $ multiset.attach_cons _ _
 
+@[simps] protected def cast {n m : ℕ} (h : n = m) : sym α n ≃ sym α m :=
+{ to_fun := λ s, ⟨s.val, s.2.trans h⟩, inv_fun := λ s, ⟨s.val, s.2.trans h.symm⟩,
+  left_inv := λ s, subtype.ext rfl, right_inv := λ s, subtype.ext rfl }
```

Comment:
Generally each field for structure notation gets its own line. The style guide claims that the `:=`'s should be aligned with each other, but that's not enforced.
```suggestion
{ to_fun := λ s, ⟨s.val, s.2.trans h⟩,
  inv_fun := λ s, ⟨s.val, s.2.trans h.symm⟩,
  left_inv := λ s, subtype.ext rfl,
  right_inv := λ s, subtype.ext rfl }
```